wikimedia/wikipedia-ios
# Add Notifications Center empty state subtitle when no projects are selected

**Phabricator:** https://phabricator.wikimedia.org/T288668 (partially)

### Notes
Adds a subtitle to the empty state when the user has no projects selected in the Notifications Center.

### Test Steps
1. Open Notifications Center > Project modal
2. Uncheck all available projects
3. Confirm empty state with "Add projects to see more messages" is shown

![IMG_0047](https://user-images.githubusercontent.com/5652327/156255906-37f33e5c-6e2f-45ae-9650-9e162cf5dc38.PNG)
